src/ddscxx/include/org/eclipse/cyclonedds/pub/qos/DataWriterQosDelegate.hpp

```diff
@@ -56,6 +56,7 @@ class OMG_DDS_API DataWriterQosDelegate
     void policy(const dds::core::policy::OwnershipStrength& strength);
 #endif  // OMG_DDS_OWNERSHIP_SUPPORT
     void policy(const dds::core::policy::WriterDataLifecycle&   lifecycle);
+    void policy(const dds::core::policy::DataRepresentation& datarepresentation);
```

Comment:
Please also encapsulate this line with the conditional OMG_DDS_EXTENSIBLE_AND_DYNAMIC_TOPIC_TYPE_SUPPORT macro.

Reply:
Will do.